clang/lib/CodeGen/CGOpenMPRuntimeGPU.cpp

```diff
@@ -3466,7 +3466,7 @@ void CGOpenMPRuntimeGPU::processRequiresDirective(
       case CudaArch::SM_20:
       case CudaArch::SM_21:
       case CudaArch::SM_30:
-      case CudaArch::SM_32:
+      case CudaArch::_SM_32:
```

Comment:
`SM_32_` ?

Reply:
Thanks, for some reason my LSP didn't rename that one.